User/user.md

```diff
@@ -0,0 +1,157 @@
+# User
+
+## Overview
+
+ A User in OpenMRS is an account that a person may use to log into the system.
+
+The real-life person is represented by a Person record in OpenMRS, and a person may have more than one user account. If you want a patient to be able to view her own record in OpenMRS, then you need to create a user account and link it to the patient.
```

Comment:
Users are only [linked to persons](https://docs.openmrs.org/datamodel/openmrs-refapp-data-model-1.9.html#users.person_id) (not directly to patients). So, it would be more correct to end this with "... then you need to create a user account and link it to the patient's person record."